data/remnant/remnant missions.txt

```diff
@@ -1061,6 +1061,334 @@ mission "Remnant: Learn Sign Follow Up"
+	npc accompany save
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+			not system "Edusa"
+		personality derelict timid plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant 7
+				Pelican
+			variant 2
+				Gull
+	on complete
+		"rescue missions" ++
+		payment 300000
+		dialog
+			`Your communicator notifies you of a resource allocation of <payment>, and also a thank you from the captain of the <npc>.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+	
+	
+mission "Remnant Rescue 3B"
+	job
+	repeat
+	name "Search and Rescue: Supply Ship"
+	description "The <npc> has gone missing during a supply run in the Ember Wastes. Find them and help them return to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		random < 3
+	npc kill
+		system
+			attributes "ember waste"
+			not attributes "ka'het"
+			not attributes "inaccessible"
+			not attributes "graveyard"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+			not system "Edusa"
+		personality staying heroic nemesis unconstrained
+		government "Test Dummy"
+		fleet
+			names "remnant"
+			variant
+				"Rescue Dummy"
+		dialog
+			`As you enter the system, your sensors pick up a lot of debris. Further analysis confirms your worst fears. The <npc> has been destroyed. You frantically spend the next few hours searching for survivors, your hope sinking with every passing minute.`
+	on complete
+		"rescue missions" ++
+		payment 300000
+		dialog
+			`You land, and with heavy heart, report that the <npc> was lost with all hands.`
+	on visit
+		dialog
+			`You have returned, yet the <npc> is still missing.`
+
+	
+mission "Remnant Rescue 4"
+	job
+	repeat
+	name "Distress Signal: Graveyard"
+	description "A distress signal has been received from the <npc> in the Graveyard. Hurry there, and help return them safely to <destination>."
+	source
+		government "Remnant"
+	to offer
+		has "Remnant: Learn Sign Follow Up: offered"
+		has "First Contact: Ka'het: Remnant 1B: offered"
+		random < 10
+	npc accompany save
+		system
+			attributes "graveyard"
+			not attributes "inaccessible"
+			not government "Remnant"
+			not system "Perfica"
+			not system "Farinus"
+			not system "Edusa"
+			not system "Aki'il"
+		personality escort derelict heroic plunders
+		government "Remnant"
+		fleet
+			names "remnant"
+			variant 8
+				Albatross
+			variant 2
+				Ibis
+	on complete
+		"rescue missions" ++
+		payment 500000
+		dialog
+			`Upon landing, the captain of the <npc> meets you at your ship to thank you personally for rescuing them from the Graveyard, leaving you with a heartfelt sentence: "May the Embers burn bright for you." `
+			`Soon after, your communicator notifies you of a resource allocation of <payment>.`
```

Comment:
```suggestion
			`Upon landing, the captain of the <npc> meets you at your ship to thank you personally for rescuing them from the Graveyard, leaving you with a heartfelt sentence: "May the Embers burn bright for you." `
			`	Soon after, your communicator notifies you of a resource allocation of <payment>.`
```
OR
```suggestion
			`Upon landing, the captain of the <npc> meets you at your ship to thank you personally for rescuing them from the Graveyard, leaving you with a heartfelt sentence: "May the Embers burn bright for you." Soon after, your communicator notifies you of a resource allocation of <payment>.`
```
Up to you. The paragraph break doesn't seem necessary to me, but if you'd like to keep it that's fine.